mezzanine/template/loader_tags.py

```diff
@@ -68,7 +72,13 @@ def find_template(self, name, context, peeking=False):
         # other loaders like the ``cached`` template loader, unwind its
         # internal loaders and add those instead.
         loaders = []
-        for loader_name in settings.TEMPLATE_LOADERS:
+        loader_names = set(chain.from_iterable(
+            [template_engine.get('OPTIONS', {}).get('loaders', [])
+             for template_engine in settings.TEMPLATES]))
```

Comment:
> Could we just use Engine.template_loaders here instead of re-parsing the settings?

Makes sense. I briefly looked for a better way to do this but didn't know where to look.